client/app/nasha/add/nasha-unavailable.controller.js

```diff
@@ -0,0 +1,14 @@
+class NashaUnavailableCtrl {
+    constructor (OvhApiMe, PhoneContact) {
+        this.OvhApiMe = OvhApiMe;
+        this.PhoneContact = PhoneContact;
+    }
+
+    $onInit () {
+        this.OvhApiMe.v6().get().$promise.then(me => {
```

Comment:
No `catch` ?

Reply:
No catch !

Reply:
We do not catch when calling /me, assuming that if /me fails we have greater problem than catch it there.
So we let angular rise its own error :)

Reply:
But then the users isn't warned 😭